wallet-health.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eo pipefail
+
+wallet_name=default
+if [[ -n $1 ]]; then
+  wallet_name=$1
+fi
+
+# this script checks if a wallet exists and is not scanning
+
+echo "Getting wallet info"
+wallet_info=$(bitcoin-cli -rpcwallet=${wallet_name} getwalletinfo)
```

Comment:
a retry logic may be useful here?

Reply:
It's a local call so it's not very likely to fail. If it does, it will just be run again on the next healthcheck interval.